value-or-json.js

```diff
@@ -21,12 +21,12 @@ function needsStringify(arg) {
     case "undefined":
     case "function":
       return false;
-      break;
```

Comment:
Looks great, just put this explicit break back in. You should always break in case statements as a future-maintenance defense mechanism. Missing breaks is nearly its own category of bugs.